lib/src/dart_dev_cli.dart

```diff
@@ -89,12 +91,6 @@ dev(List<String> args) async {
 }
 
 void registerTask(TaskCli cli, TaskConfig config) {
-  if (_cliTasks[cli.command] != null) {
-    throw new ArgumentError(
```

Comment:
Mm, or perhaps we should not throw when a local task overrides a built-in one, and warn in all other cases?